docs/codex/README.md

```diff
@@ -21,7 +21,7 @@ For more information, see the [Antibodies TSV documentation](../antibodies).
 
 | pattern | example | required? | description |
 | --- | --- | --- | --- |
-| `NAV[^/]+\.tif` |  |  | Navigational Image showing Region of Interest (Keyance Microscope only) |
+| `[^/]+NAV[^/]+\.tif` |  |  | Navigational Image showing Region of Interest (Keyance Microscope only) |
```

Comment:
@mccalluc I am trying to match all files that include `NAV`. You are right I mean `*`. Since you are the owner it always adds you as a reviewer, sorry. Thanks for the suggestion, you are right I was wrong.